scripts.py

```diff
@@ -3148,7 +3139,15 @@ def add_line(id, name, protocol, metadata_identifier):
 
 
 class UpdateLaneSizeScript(LaneSweeperScript):
+
+    def should_process_lane(self, lane):
+        """We don't want to process generic WorkLists -- there's nowhere
+        to store the data.
+        """
+        return isinstance(lane, Lane)
+
```

Comment:
Without this, `bin/update_lane_size` starts off trying to process the top-level WorkList for a library, and it crashes immediately.